Dolphin scripts/Entrance Randomizer/lib/entrance_rando.py

```diff
@@ -352,69 +505,102 @@ def get_random_redirection(original: Transition, all_redirections: Iterable[Tran
 
 def set_transitions_map():  # noqa: PLR0915 # TODO: Break up in smaller functions
```

Comment:
Well yeah I can break it up as much as you want, but weren't you the one who wanted merge requests to be as small as possible?
If you want me to overhaul this python file even more than I already have, and you want me to do so in this very same MR... then sure I'll do that.
I would just like to get your confirmation on this part.

Reply:
The best way to keep PRs self-contained, would be to have a separate PR first that split it off.
Since you're already well started on this one, *if* we agree to have a small PR that does nothing else than split up this function right after it's merged, then you can add.
```suggestion
# TODO: Break up in smaller functions before changing anything else
def set_transitions_map():  # noqa: C901, PLR0912, PLR0914, PLR0915
```

Reply:
Sure, I can agree to that.